docs/deployment/teams.md

```diff
@@ -53,12 +53,25 @@ Members can't view or perform any actions in the following pages:
 - Projects
 - Settings
 
+### Invite a team member through email
+
+Add team members easily to your organization by sending them an email. This email contains instructions for them to join your org through the same auth provider configured for your account.
+
+You must be an `admin` to perform this operation.
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep Cloud Platform](https://semgrep.dev/login).
+1. Click **<i class="fa-solid fa-gear"></i> Settings > Access**. This brings you to the **Members** tab.
+1. Click **Invite members**.
+1. In the dialog, enter your team members' email addresses. You can invite up to 20 at a time. Separate each email address with a <kbd>Space</kbd> or <kbd>Tab</kbd> key.
```

Comment:
```suggestion
1. In the dialog, enter your team members' email addresses. You can invite up to 20 users at a time. Separate each email address with a <kbd>Space</kbd> or <kbd>Tab</kbd> key.
```

Reply:
You can also paste in a comma separated list of emails (e.g `a@a.com, b@b.com, c@c.com`) into the form